### PR TITLE
Student Tracker page done

### PR DESCRIPTION
**Pull request by Denis-Andrei**
This PR includes:
- svg icon colors modified
- following components created: Header, Heading, SideBar, BoxTracker
- added some mixins and variables in main.scss

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Front from './pages/Front/Front';
+import {Route} from 'react-router-dom';
+import Login from './pages/Login/Login';
+import StudentTracker from './pages/StudentTracker/StudentTracker';
+
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
```
Comment (Wolfjawan) [inline src/App.js:11]: Would be better if we move the routes into  a new folder in src folder and name it routes

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <BrowserRouter>
```
Comment (Wolfjawan) [inline src/index.js]: Also would be good if move BrowserRouter one level dipper into app file or routes

**Review by Wolfjawan — CHANGES_REQUESTED** [review]
Looks good but needs some improvements

File: src/index.js (hunk 2 above)
Comment (ISTANBULBEKLE) [reply src/index.js]: In line with what Mohsen noted, may I also suggest having a general understanding of how we built file structure; Components, Components CSS files, images and icons container etc.?

**Review by ISTANBULBEKLE — APPROVED** [review]
Thanks a lot, team. Great effort. I have some question about SVG files, SASS use and folder structures. We will discuss it later.

**Review by ISTANBULBEKLE — APPROVED** [review]

**Review by mohsen-moradi-cg — APPROVED** [review]
LGTM 👍

**Review by Wolfjawan — APPROVED** [review]
LGTM 👍

**Pull request merged**